pkg/sensors/tracing/selectors.go

```diff
@@ -571,3 +574,37 @@ func populateStringPostfixFilterMap(
 
 	return nil
 }
+
+func validateActionSelectors(probes []v1alpha1.KProbeSelector) error {
+	for _, p := range probes {
+		selectors := p.MatchActions[:]
+		if p.MatchReturnActions != nil {
+			selectors = append(selectors, p.MatchReturnActions...)
+		}
+		for _, selector := range selectors {
+			switch strings.ToLower(selector.Action) {
+			case "sigkill":
+				if option.Config.DisableSignalActions {
+					return fmt.Errorf("sigkill actions are disabled")
+				}
+			case "signal":
+				if option.Config.DisableSignalActions {
+					return fmt.Errorf("signal actions are disabled")
+				}
```

Comment:
nit: combine these two cases into a single case since they seem to be checking the same thing.

Reply:
I would also say, would we want to group these two options so that we don't end up with "too many flags"? Is it very likely that someone that would use one would like to use the other?

Reply:
I just meant that  having something like:
> case "sigkill", "signal":
> ...